.github/workflows/black.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.11"
```

Comment:
Statina require >=3.6
Genotype API >= 3.10
Genotype = 3.6 
Not sure what the effect will be on the genotype repo.